ckb/src/main/java/org/nervos/ckb/type/transaction/TransactionWithStatus.java

```diff
@@ -4,6 +4,9 @@
 
 /** Copyright © 2019 Nervos Foundation. All rights reserved. */
 public class TransactionWithStatus {
+  public static final String PENDING = "pending";
```

Comment:
suggest creating an enum for status

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/252/commits/c2871cfe874d0ca6966309dcc2d43afd2d145421